Cargo.toml

```diff
@@ -13,11 +13,13 @@ keywords = ["encryption", "xts"]
 exclude = ["test_files"]
 
 [features]
+default = ["std"]
+std = ["byteorder/std", "cipher/std"]
 openssl_tests = ["openssl"]
 
 [dependencies]
-cipher = "0.4"
-byteorder = "1"
+cipher = { version = "0.4", default-features = false }
```

Comment:
I don't think we need to disable `default-features` as cipher doesn't enable `std` by default